README.md

```diff
@@ -523,12 +524,38 @@ I would like to hear from you in which scenario you are using ha-sip!
 3. Copy `.env.example` to `.env` and replace the variable place-holders with your real configuration.
 
    `HA_BASE_URL` is something like "http://homeassistant.local:8123/api"
-   
-   The access token is created from http://homeassistant.local:8123/profile 
+
+   The access token is created from http://homeassistant.local:8123/profile
 4. Run `./build.sh run-local` to run the add-on locally
 5. Paste commands as json (without line-breaks) into stdin of the running add-on:
 
-   Example: 
+   Example:
    ```json
    { "command": "dial", "number": "sip:**620@fritz.box", "menu": { "message": "Hello from ha-sip.", "language": "en" } }
    ```
+
+## Stand-alone mode
+
+The stand-alone mode can be used if you run home assistant in a docker enviornment and you don't have access to the hassio.addon_stdin service. Instead of stdin - mqtt will be used for communication.
+
+1. Follow the instructions from home assistant to set up a working MQTT broker and install the MQTT integration [MQTT Broker](https://www.home-assistant.io/integrations/mqtt/)
+2. Copy `.env.example` to `.env` and replace the variable place-holders with your real configuration.
+3. Make sure you switched the COMMAND_SOURCE in your .env file from "stdin" to "mqtt" and set the BROKER_* variables to connect to your MQTT broker address
+4. Install [docker-compose](https://docs.docker.com/compose/install/) (Hint: Eventhough we just run one service, a simple docker run --env-file won't work, since it's not un-quoting the .env variables correctly)
```

Comment:
docker-compose is deprecated. It's `docker compose` now. See https://docs.docker.com/compose/migrate/